backstory/src/main/webapp/style.css

```diff
@@ -1,3 +1,17 @@
+/* Color Scheme */
+:root {
```

Comment:
While I wish we could use these, we should aim to use valid conventions that will satisfy master merges for the MVP.

Reply:
As we discussed offline, due to new CSS validator, we should be able to use CSS constants.